app/routes/__index/quest/sign/$user/$tokenId.tsx

```diff
@@ -0,0 +1,38 @@
+import { ec } from "starknet";
+import { pedersen } from "starknet/dist/utils/hash";
+import { sign } from "starknet/dist/utils/ellipticCurve";
+
+import eligibleUsers from "../../../../../../public/eligibleUsers.json";
+import type { LoaderArgs } from "@remix-run/node";
+
+
+export async function loader({ params }: LoaderArgs) {
+    const user = params.user
+    const privateKey = process.env.QUEST_SIGNER_PRIVATE_KEY;
```

Comment:
Make sure to add a note in the ticket to let @cloudvenger know that env variables need to be set when deploying on fly.

Reply:
where to leave the note ? in #68 ?